src/Script/RunningScript.ts

```diff
@@ -106,7 +106,8 @@ export class RunningScript {
 
     let logEntry = txt;
     if (Settings.TimestampsFormat && typeof txt === "string") {
-      logEntry = "[" + formatTime(Settings.TimestampsFormat) + "] " + String(logEntry);
+      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
Optional: Since you disabled the lint check for template string, by using template format here too you can skip the operand suppression.

Reply:
Using template literals here will trigger "no-base-to-string".

Reply:
Yes, that is correct. The linter is highlighting an issue.

this.logs is an array of react nodes, not strings. If you want literal text, these need to be converted to JSX. If you want string[], the logs need to be typed differently.

Reply:
```suggestion
    let logEntry = txt;
    if (Settings.TimestampsFormat && typeof txt === "string") {
      logEntry = `[${formatTime(Settings.TimestampsFormat)}] ${txt}`;
    }

    this.logs.push(logEntry);
```

Reply:
At first, I'm sorry if I sound rude here. That's not my intention. This is the same case as your previous comment of `${opts}`. I read your message, and I don't get what you want.

This is just my opinion, so feel free to skip it. When I read a code review, I want to know exactly what the reviewer wants. If everything is good, there is no need to add a review; we can just merge. If there is a review, it means that something must/should be changed. In that case, I appreciate this type of review:
- I disagree with your X, because I think Y is better.
- Suggestion: Do X instead of Y because Z.
- I'll veto this change due to X.
- etc.

The key thing here is the "action". I want to know the expected "action" of the reviewer. It's normal to disagree with each other, so your expected "action" may not be what I want to do, but I need to know what it is before I can think about it. d0sboots's review is usually excellent in this matter. They always tell me clearly what they want to change ("You do X, but I want Y"). Having extended discussion ("food for thought") is totally fine, but it should come after that.

Back to this review, what do you want me to change?

Reply:
No worries :)